__tests__/AppCatalog.js

```diff
@@ -22,162 +26,633 @@ import {
+      }
+    );
+    const { findByText } = renderRouteWithStore(appCatalogListPath);
+
+    // The app's description should be there.
+    // This comes from parsing the index.yaml, which is mocked in catalogIndexResponse.
+    const appDescription = await findByText(
+      'A Helm chart for the nginx ingress-controller'
+    );
+    expect(appDescription).toBeInTheDocument();
+  });
+
+  it('installs an app in a cluster, with default settings', async () => {
+    const installAppResponse = {
+      code: 'RESOURCE_CREATED',
+      message: `We're installing your app called 'nginx-ingress-controller-app' on ${V4_CLUSTER.id}`,
+    };
+    const installAppRequest = nock(API_ENDPOINT)
+      .intercept(
+        `/v4/clusters/${V4_CLUSTER.id}/apps/nginx-ingress-controller-app/`,
+        'PUT'
+      )
+      .reply(StatusCodes.Ok, installAppResponse);
+
+    // eslint-disable-next-line no-magic-numbers
+    getMockCall('/v4/appcatalogs/', appCatalogsResponse);
+    getMockCall(`/v4/organizations/${ORGANIZATION}/credentials/`);
+    getMockCall(`/v4/clusters/${V4_CLUSTER.id}/`, v4AWSClusterResponse);
+    getMockCall(
+      `/v4/clusters/${V4_CLUSTER.id}/status/`,
+      v4AWSClusterStatusResponse
+    );
+    getMockCall(`/v4/clusters/${V4_CLUSTER.id}/apps/`, appsResponse);
+    getMockCall(`/v4/clusters/${V4_CLUSTER.id}/key-pairs/`);
+    getMockCall('/v4/clusters/', v4ClustersResponse);
+    getMockCall('/v4/releases/', releasesResponse);
+
+    nock('https://catalogshost')
+      .get('/giantswarm-incubator-catalog/index.yaml')
+      .reply(StatusCodes.Ok, catalogIndexResponse);
+    nock('https://catalogshost')
+      .get('/giantswarm-test-catalog/index.yaml')
+      .reply(StatusCodes.Ok, catalogIndexResponse);
+    nock('https://catalogshost')
+      .get('/helmstable/index.yaml')
+      .reply(StatusCodes.Ok, catalogIndexResponse);
+
+    const appCatalogListPath = RoutePath.createUsablePath(
+      AppCatalogRoutes.AppDetail,
+      {
+        repo: 'giantswarm-incubator',
+        app: 'nginx-ingress-controller-app',
+      }
+    );
+    const { findByText, findAllByText, getByText } = renderRouteWithStore(
+      appCatalogListPath
+    );
+
+    // Press the configure button
+    let installButton = await findByText(/configure & install/i);
+    fireEvent.click(installButton);
+
+    // Select the current cluster
+    fireEvent.click(getByText(V4_CLUSTER.name));
+
+    // Install the new app
+    installButton = await findByText(/install app/i);
+    fireEvent.click(installButton);
+
+    await findByText(/is being installed on/i);
+
+    // Check if the user got redirected to the cluster detail view
+    await findAllByText(V4_CLUSTER.name);
+    await findByText(/kubernetes endpoint uri/i);
+
+    /**
+     * Manually removing the toast notification to prevent
+     * it from showing up in other tests
+     */
+    document.querySelector('#noty_layout__topRight').remove();
+
+    installAppRequest.done();
```

Comment:
`.done()` on a nock scope doesn't actually do anything. It returns true or false based on whether the interceptor has intercepted a call or not. It can be used in an assertion, but on it's own, no need for it.